azure-pipelines.yml

```diff
@@ -103,29 +127,53 @@ jobs:
 
   - script: |
       export LLVM_CONFIG_PATH=$(brew --prefix llvm)/bin/llvm-config
-      ./scripts/run_ci_checks.sh fmt
+      export RUSTFLAGS="-D warnings"
+      export RUSTDOCFLAGS="-D warnings"
+      cargo fmt --check
     displayName: 'cargo fmt --check'
   
   - script: |
       export LLVM_CONFIG_PATH=$(brew --prefix llvm)/bin/llvm-config
-      ./scripts/run_ci_checks.sh check
+      export RUSTFLAGS="-D warnings"
+      export RUSTDOCFLAGS="-D warnings"
+      # Soon to be superceded by the commented out `cargo clippy` below.
+      # This is different from `cargo build`
+      # as this uses `--all-features` to check everything.
+      cargo check --tests --all-features
+      # cargo clippy --tests --all-features
     displayName: 'cargo check'
 
   - script: |
```

Comment:
I think it is safe to skip rustdoc on Darwin since we're mostly interested in detecting whether we broke the build or any tests.

Reply:
See https://github.com/immunant/c2rust/pull/696#issuecomment-1272655223.